lib/VeresOneDidDoc.js

```diff
@@ -68,10 +68,10 @@ class VeresOneDidDoc {
    * as well as authentication and authorization keys.
    * Only called when generating a new DID Doc (creates new keys, etc).
    *
-   * @param [mode] {string} 'dev' / 'test' etc.
+   * @param [mode] - - {string} 'dev' / 'test' etc.
```

Comment:
what's going on with the dashes here and below?

Reply:
this --fix it looks like when the doc strings are written incorrectly i.e. name then type it does not recognize them.

Reply:
your comment here suggests to me that you did not review this diff yourself before submitting the PR, is that correct?

Reply:
this PR is actually just supposed to be the eslint upgrade. I was checking to make sure the 2.0.0 lint works with this (which means the .eslintrc.js file in this commit is really strange because I'm pretty confident I added the base rules). Anyways, I did check the .js file diff just to make sure all that was changed was the doc strings. I see your point and will definitely review the --fix changes and see what else it messed up.

Reply:
I was planning on the doc update after lint update! but let's just do it in this PR.